src/xo-app/vm/tab-disks.js

```diff
@@ -452,6 +516,32 @@ export default class TabDisks extends Component {
       bootOrder,
       newDisk
     } = this.state
+    const userData = {srs, vm, vbds}
+    const INDIVIDUAL_ACTIONS = [
```

Comment:
Move this definition out of the render function (for instance using [field declarations](https://github.com/tc39/proposal-class-fields#field-declarations) syntax).